tests/unit/job/test_query.py

```diff
@@ -970,7 +877,12 @@ def test_result(self):
             "rows": [{"f": [{"v": "abc"}]}],
         }
         conn = make_connection(
-            query_resource, query_resource_done, job_resource_done, query_page_resource
+            job_resource,
```

Comment:
Not sure I am tracking the relationship between the make connection inputs versus the assert_has_calls checks.

Can you explain how these tests are supposed to work?